django_netjsonconfig/base/admin.py

```diff
@@ -60,6 +60,8 @@ def get_extra_context(self, pk=None):
         }
         if pk:
             ctx['download_url'] = reverse('{0}_download'.format(prefix), args=[pk])
+            if self.model.__name__ == 'Device' and not self.model.objects.get(pk=pk)._has_config():
```

Comment:
try to drop an `import ipdb; ipdb.set_trace()` and check if there's anything like `self.instance` or similar with the instance already preloaded

Reply:
@nemesisdesign No instance provided.